<launch>

    <node pkg="raspicam_node" type="raspicam_node" name="camera" output="screen">

        <param name="camera_info_url" value="package://raspicam_node/camera_info/camerav2_1280x720.yaml"/>
        <param name="camera_name" value="front_cam"/>
        <param name="width" value="320"/>
        <param name="height" value="240"/>
        <!-- We are running at 90fps to reduce motion blur -->
        <param name="framerate" value="90"/>
        <param name="vFlip" value="true"/>
        <param name="camera_frame_id" value="front_cam"/>

    </node>
    <node pkg="robocar" type="laser_node" name="laser">

        <param name="freq_hz" value="20" />

    </node>
</launch>